tstyche.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://tstyche.org/schemas/config.json",
+  "checkSourceFiles": true,
+  "rejectAnyType": true,
+  "testFileMatch": [
+    "packages/*/dtslint/**/*.tst.*"
```

Comment:
After running `pnpm build` lots of files are emitted. This pattern does not walk into `build`, `dist`, `src` or `test` directories. On my computer adding this pattern save around `0.3` seconds at startup of TSTyche.